src/reporter/subscription_builder.py

```diff
@@ -0,0 +1,29 @@
+from utils.subscription_dsl import *
+
+
+def build_subscription(quantumleap_url,
```

Comment:
not for now, but this does not work if the end point is protected. we need at least a way to pass a token over the header (and header parameter may be different depending on how the context broker end point is protected)

Reply:
Oh right, didn't know about that! But it's easy to change I reckon. Will need your help to understand better how it should work though.

Reply:
it's more a todo, not worries. in general we have to think about this auth aspects not only for consuming data, but also in case we interact with other APIs.